bluez/src/device.c

```diff
@@ -2578,6 +2578,21 @@ void device_bonding_complete(struct btd_device *device, uint8_t status)
 	if (status) {
 		device_cancel_authentication(device, TRUE);
 		device_cancel_bonding(device, status);
+
+		if (status == 0x06 && device_is_paired(device)) {
```

Comment:
Yes. Upstream code often uses numeric constants but no reason why we should follow that all the way. Updating PR.